pygls/protocol.py

```diff
@@ -369,8 +370,12 @@ def _send_data(self, data):
         if not data:
             return
 
+        def default(o):
```

Comment:
Could you just move this function under `call_user_feature`(line 70.) and perhaps rename it to `default_serializer` or something similar?

Reply:
Thanks, good idea, I pushed more fixup commits to address this.